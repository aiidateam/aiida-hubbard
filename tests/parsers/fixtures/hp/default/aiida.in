&INPUTHP
  conv_thr_chi =   1.0000000000d-06
  iverbosity = 2
  nq1 = 1
  nq2 = 1
  nq3 = 1
  outdir = 'out'
  prefix = 'aiida'
/
