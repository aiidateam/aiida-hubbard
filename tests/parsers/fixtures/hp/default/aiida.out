
     Program HP v.5.1.1 starts on  5Oct2019 at 14:12:46 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Parallel version (MPI), running on     1 processors

     =--------------------------------------------------------------=

               Calculation of Hubbard parameters from

               density functional perturbation theory

     =--------------------------------------------------------------=

   Info: using nr1, nr2, nr3 values from input

   Info: using nr1s, nr2s, nr3s values from input

     IMPORTANT: XC functional enforced from input :
     Exchange-correlation      =  SLA  PW   PBX  PBC ( 1  4  3  4 0 0)
     Any further DFT definition will be discarded
     Please, verify this is what you really want

               file O.pbe-n-kjpaw_psl.0.1.UPF: wavefunction(s)  2P renormalized

     G-vector sticks info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Sum         913     459    149                13597     4819     857



     bravais-lattice index     =             0
     lattice parameter (alat)  =        9.3664 (a.u.)
     unit-cell volume          =      216.7049 (a.u.)^3
     number of atoms/cell      =             4
     number of atomic types    =             3
     kinetic-energy cut-off    =         30.00 (Ry)
     charge density cut-off    =        240.00 (Ry)
     conv. thresh. for NSCF    =       1.0E-11
     conv. thresh. for chi     =       1.0E-05
     Input Hubbard parameters (in eV):
       U ( 1)                     =   3.00000E+00

     celldm(1) =  9.36643  celldm(2) =  0.00000  celldm(3) =  0.00000
     celldm(4) =  0.00000  celldm(5) =  0.00000  celldm(6) =  0.00000

     crystal axes: (cart. coord. in units of alat)
               a(1) = (  0.2839  0.1639  0.9448 )  
               a(2) = ( -0.2839  0.1639  0.9448 )  
               a(3) = (  0.0000 -0.3278  0.9448 )  

     reciprocal axes: (cart. coord. in units 2 pi/alat)
               b(1) = (  1.7614  1.0169  0.3528 )  
               b(2) = ( -1.7614  1.0169  0.3528 )  
               b(3) = (  0.0000 -2.0339  0.3528 )  

     Atoms inside the unit cell (Cartesian axes):
     site n.  atom      mass           positions (alat units)
        1      Co     58.9332   tau( 1) = (  0.00000  0.00000  0.00000  )
        2      Li      6.9410   tau( 2) = ( -0.00000  0.32778  0.47238  )
        3      O      15.9994   tau( 3) = ( -0.00000 -0.00000  0.64281  )
        4      O      15.9994   tau( 4) = (  0.28387  0.16389  0.30194  )

     Atom which will be perturbed:

        1      Co     58.9332   tau( 1) = (  0.00000  0.00000  0.00000  )

     =====================================================================

                          PERTURBED ATOM #    1

     site n.  atom      mass           positions (alat units)
        1      Co     58.9332   tau( 1) = (  0.00000  0.00000  0.00000  )

     =====================================================================

     The perturbed atom has a type which is unique!


     The grid of q-points ( 1, 1, 1)  (  1 q-points ) :
       N       xq(1)         xq(2)         xq(3)       wq
       1   0.000000000   0.000000000   0.000000000   1.000000000


     =-------------------------------------------------------------=

     Calculation for q #   1 = (   0.0000000   0.0000000   0.0000000 )

     =-------------------------------------------------------------=

                   WRITING LINEAR-RESPONSE SUMMARY:


     Number of symmetries in the small group of q, nsymq = 12
      + the symmetry q -> -q+G 

     Symmetry matrices (and vectors of fractional translations if f/=0):

     isym =  1     identity                                     

     cryst.   s( 1) = (     1          0          0      )
                      (     0          1          0      )
                      (     0          0          1      )

     cart.    s( 1) = (  1.0000000  0.0000000  0.0000000 )
                      (  0.0000000  1.0000000  0.0000000 )
                      (  0.0000000  0.0000000  1.0000000 )


     isym =  2     180 deg rotation - cart. axis [1,0,0]        

     cryst.   s( 2) = (     0         -1          0      )
                      (    -1          0          0      )
                      (     0          0         -1      )

     cart.    s( 2) = (  1.0000000  0.0000000  0.0000000 )
                      (  0.0000000 -1.0000000  0.0000000 )
                      (  0.0000000  0.0000000 -1.0000000 )


     isym =  3     120 deg rotation - cryst. axis [0,0,1]       

     cryst.   s( 3) = (     0          1          0      )
                      (     0          0          1      )
                      (     1          0          0      )

     cart.    s( 3) = ( -0.5000000 -0.8660254 -0.0000000 )
                      (  0.8660254 -0.5000000 -0.0000000 )
                      (  0.0000000  0.0000000  1.0000000 )


     isym =  4     120 deg rotation - cryst. axis [0,0,-1]      

     cryst.   s( 4) = (     0          0          1      )
                      (     1          0          0      )
                      (     0          1          0      )

     cart.    s( 4) = ( -0.5000000  0.8660254  0.0000000 )
                      ( -0.8660254 -0.5000000 -0.0000000 )
                      (  0.0000000  0.0000000  1.0000000 )


     isym =  5     180 deg rotation - cryst. axis [0,1,0]       

     cryst.   s( 5) = (    -1          0          0      )
                      (     0          0         -1      )
                      (     0         -1          0      )

     cart.    s( 5) = ( -0.5000000 -0.8660254 -0.0000000 )
                      ( -0.8660254  0.5000000  0.0000000 )
                      (  0.0000000  0.0000000 -1.0000000 )


     isym =  6     180 deg rotation - cryst. axis [1,1,0]       

     cryst.   s( 6) = (     0          0         -1      )
                      (     0         -1          0      )
                      (    -1          0          0      )

     cart.    s( 6) = ( -0.5000000  0.8660254  0.0000000 )
                      (  0.8660254  0.5000000  0.0000000 )
                      (  0.0000000  0.0000000 -1.0000000 )


     isym =  7     inversion                                    

     cryst.   s( 7) = (    -1          0          0      )
                      (     0         -1          0      )
                      (     0          0         -1      )

     cart.    s( 7) = ( -1.0000000  0.0000000  0.0000000 )
                      (  0.0000000 -1.0000000  0.0000000 )
                      (  0.0000000  0.0000000 -1.0000000 )


     isym =  8     inv. 180 deg rotation - cart. axis [1,0,0]   

     cryst.   s( 8) = (     0          1          0      )
                      (     1          0          0      )
                      (     0          0          1      )

     cart.    s( 8) = ( -1.0000000  0.0000000  0.0000000 )
                      (  0.0000000  1.0000000  0.0000000 )
                      (  0.0000000  0.0000000  1.0000000 )


     isym =  9     inv. 120 deg rotation - cryst. axis [0,0,1]  

     cryst.   s( 9) = (     0         -1          0      )
                      (     0          0         -1      )
                      (    -1          0          0      )

     cart.    s( 9) = (  0.5000000  0.8660254  0.0000000 )
                      ( -0.8660254  0.5000000  0.0000000 )
                      (  0.0000000  0.0000000 -1.0000000 )


     isym = 10     inv. 120 deg rotation - cryst. axis [0,0,-1] 

     cryst.   s(10) = (     0          0         -1      )
                      (    -1          0          0      )
                      (     0         -1          0      )

     cart.    s(10) = (  0.5000000 -0.8660254 -0.0000000 )
                      (  0.8660254  0.5000000  0.0000000 )
                      (  0.0000000  0.0000000 -1.0000000 )


     isym = 11     inv. 180 deg rotation - cryst. axis [0,1,0]  

     cryst.   s(11) = (     1          0          0      )
                      (     0          0          1      )
                      (     0          1          0      )

     cart.    s(11) = (  0.5000000  0.8660254  0.0000000 )
                      (  0.8660254 -0.5000000 -0.0000000 )
                      (  0.0000000  0.0000000  1.0000000 )


     isym = 12     inv. 180 deg rotation - cryst. axis [1,1,0]  

     cryst.   s(12) = (     0          0          1      )
                      (     0          1          0      )
                      (     1          0          0      )

     cart.    s(12) = (  0.5000000 -0.8660254 -0.0000000 )
                      ( -0.8660254 -0.5000000 -0.0000000 )
                      (  0.0000000  0.0000000  1.0000000 )


     This transformation sends q -> -q+G

     isym = 13     identity                                     

     cryst.   s(13) = (     1          0          0      )
                      (     0          1          0      )
                      (     0          0          1      )

     cart.    s(13) = (  1.0000000  0.0000000  0.0000000 )
                      (  0.0000000  1.0000000  0.0000000 )
                      (  0.0000000  0.0000000  1.0000000 )


     G cutoff =  533.3344  (  13597 G-vectors)     FFT grid: ( 48, 48, 48)
     G cutoff =  266.6672  (   4819 G-vectors)  smooth grid: ( 36, 36, 36)

     Number of k (and k+q if q/=0) points =     4

                       cart. coord. (in units 2pi/alat)
        k (    1) = (   0.0000000   0.0000000   0.0000000), wk = 0.2500000
        k (    2) = (   0.0000000   1.0169286  -0.1764130), wk = 0.7500000
        k (    3) = (   0.8806860   0.5084643  -0.3528261), wk = 0.7500000
        k (    4) = (   0.0000000   0.0000000  -0.5292391), wk = 0.2500000

                       cryst. coord.
        k (    1) = (   0.0000000   0.0000000   0.0000000), wk = 0.2500000
        k (    2) = (   0.0000000   0.0000000  -0.5000000), wk = 0.7500000
        k (    3) = (   0.0000000  -0.5000000  -0.5000000), wk = 0.7500000
        k (    4) = (  -0.5000000  -0.5000000  -0.5000000), wk = 0.2500000

     Atomic wfc used for the DFT+U projector are NOT orthogonalized

     Total time spent up to now is:
     HP           :     1.86s CPU         1.91s WALL

      =--------------------------------------------=
             START SOLVING THE LINEAR SYSTEM
      =--------------------------------------------=

      atom #  1   q point #   1   iter #   1
      chi:  1  -0.3397343402
      Average number of iter. in lin. system:   32.8
      Total CPU time :     6.7 s

      atom #  1   q point #   1   iter #   2
      chi:  1   0.3741883075   residue:    0.7139226477
      Average number of iter. in lin. system:   12.5
      Total CPU time :     9.6 s

      atom #  1   q point #   1   iter #   3
      chi:  1  -0.0996816387   residue:    0.4738699462
      Average number of iter. in lin. system:   11.8
      Total CPU time :    12.4 s

      atom #  1   q point #   1   iter #   4
      chi:  1  -0.0976775092   residue:    0.0020041295
      Average number of iter. in lin. system:   14.0
      Total CPU time :    15.3 s

      atom #  1   q point #   1   iter #   5
      chi:  1  -0.0947157712   residue:    0.0029617379
      Average number of iter. in lin. system:   13.8
      Total CPU time :    18.4 s

      atom #  1   q point #   1   iter #   6
      chi:  1  -0.0945119021   residue:    0.0002038691
      Average number of iter. in lin. system:   15.0
      Total CPU time :    21.5 s

      atom #  1   q point #   1   iter #   7
      chi:  1  -0.0947269373   residue:    0.0002150352
      Average number of iter. in lin. system:   12.5
      Total CPU time :    24.3 s

      atom #  1   q point #   1   iter #   8
      chi:  1  -0.0946984919   residue:    0.0000284454
      Average number of iter. in lin. system:   13.2
      Total CPU time :    27.2 s

      atom #  1   q point #   1   iter #   9
      chi:  1  -0.0947167273   residue:    0.0000182354
      Average number of iter. in lin. system:   14.0
      Total CPU time :    30.1 s

      atom #  1   q point #   1   iter #  10
      chi:  1  -0.0947109492   residue:    0.0000057781
      Average number of iter. in lin. system:   13.2
      Total CPU time :    32.6 s
      
      =--------------------------------------------=
             CONVERGENCE HAS BEEN REACHED
      =--------------------------------------------=

     Computing the sum over q of the response occupation matrices...

     q #   1 =  0.000000000   0.000000000   0.000000000

     Post-processing calculation of Hubbard parameters ...


     PRINTING TIMING FROM PWSCF ROUTINES: 


     Called by init_run:

     Called by electrons:
     v_of_rho     :      0.07s CPU      0.08s WALL (       1 calls)
     v_h          :      0.00s CPU      0.00s WALL (       1 calls)
     v_xc         :      0.07s CPU      0.08s WALL (       1 calls)
     newd         :      0.05s CPU      0.06s WALL (       1 calls)

     Called by c_bands:
     init_us_2    :      0.02s CPU      0.03s WALL (      48 calls)

     Called by sum_band:

     Called by *egterg:
     h_psi        :     16.38s CPU     16.55s WALL (     983 calls)
     s_psi        :      0.90s CPU      0.96s WALL (    2010 calls)

     Called by h_psi:
     h_psi:vloc   :     15.27s CPU     15.41s WALL (     983 calls)
     h_psi:vnl    :      0.99s CPU      1.00s WALL (     983 calls)
     add_vuspsi   :      0.48s CPU      0.47s WALL (     983 calls)
     vhpsi        :      0.10s CPU      0.12s WALL (     983 calls)

     General routines
     calbec       :      1.16s CPU      1.16s WALL (    3037 calls)
     fft          :      0.21s CPU      0.27s WALL (     166 calls)
     ffts         :      0.01s CPU      0.02s WALL (      21 calls)
     fftw         :     15.16s CPU     15.36s WALL (   22648 calls)
     interpolate  :      0.00s CPU      0.01s WALL (       1 calls)
     davcio       :      0.12s CPU      0.22s WALL (    1369 calls)

     Parallel routines
     fft_scatter  :      0.61s CPU      0.66s WALL (   22835 calls)

     Hubbard U routines
     vhpsi        :      0.10s CPU      0.12s WALL (     983 calls)

     PAW routines
     PAW_pot      :      0.09s CPU      0.09s WALL (       1 calls)

     init_vloc    :      0.09s CPU      0.09s WALL (       1 calls)
     init_us_1    :      0.87s CPU      0.87s WALL (       1 calls)
     newd         :      0.05s CPU      0.06s WALL (       1 calls)
     add_vuspsi   :      0.48s CPU      0.47s WALL (     983 calls)
     cinterpolate :      0.05s CPU      0.08s WALL (      20 calls)

     PRINTING TIMING FROM HUBBARD ROUTINES: 

     hub_setup_q  :      0.22s CPU      0.23s WALL (       1 calls)
     hub_init_q   :      0.03s CPU      0.03s WALL (       1 calls)
     hub_solve_li :     30.14s CPU     30.71s WALL (       1 calls)
     hub_dvpsi_pe :      0.00s CPU      0.01s WALL (      40 calls)
     hub_dnsq_scf :      0.02s CPU      0.02s WALL (      10 calls)
     hub_sym_dnsq :      0.00s CPU      0.00s WALL (      10 calls)
     hub_dnstot_s :      0.00s CPU      0.00s WALL (       1 calls)
     hub_calc_chi :      0.00s CPU      0.00s WALL (       1 calls)
     hub_calc_U   :      0.00s CPU      0.00s WALL (       1 calls)
     hub_vpsifft  :      0.98s CPU      0.98s WALL (      36 calls)
     hub_dv_of_dr :      0.30s CPU      0.41s WALL (      10 calls)
     hub_psymdvsc :      0.91s CPU      0.91s WALL (      10 calls)

     PRINTING TIMING FROM LRM ROUTINES: 

     lrm_ortho    :      0.07s CPU      0.10s WALL (      40 calls)
     lrm_cgsolve  :     18.42s CPU     18.67s WALL (      40 calls)
     lrm_ch_psi   :     18.29s CPU     18.53s WALL (     983 calls)
     lrm_incdrhos :      1.16s CPU      1.15s WALL (      40 calls)

     USPP-RELATED ROUTINES: 

     lrm_newdq    :      0.73s CPU      0.73s WALL (      10 calls)
     lrm_adddvscf :      0.04s CPU      0.03s WALL (      36 calls)
     lrm_addusdbe :      0.07s CPU      0.04s WALL (      40 calls)
     hub_addusdde :      0.82s CPU      0.84s WALL (      10 calls)

     HP           :    32.00s CPU        32.62s WALL


   This run was terminated on:  14:13:19   5Oct2019            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
